integration_tests/e2e/refer.cy.ts

```diff
@@ -40,4 +40,20 @@ context('General Refer functionality', () => {
       Page.verifyOnPage(NewReferralCompletePage)
     })
   })
+
+  describe('When a draft referral attempts to be viewed by a user other than the referrer user', () => {
```

Comment:
`When a user attempts to view a draft referral and they are not the referrer user`? "draft referral attempts" sounds like the referral is sentient!